src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200910135857_Routine_RemoveSoftDeletedSubjects.sql

```diff
@@ -0,0 +1,5 @@
+CREATE OR ALTER PROCEDURE RemoveSoftDeletedSubjects
+AS
+BEGIN
+    DELETE FROM Subject WHERE SoftDeleted = 1;
```

Comment:
Since this is a fairly simple command, was there no way built in to do this without a stored procedure?

Reply:
sadly not!